### DISCUSSION TITLE
Budibase Table

### DISCUSSION THREAD

**Discussion by ozannsk** [Budibase/budibase#5197] [Help]
Hi,

I have a few questions about budibase table:

1- My query's result is shown as a table. However the width of the table does not fit. So we need to slide to the right in order to see the rest of the columns. How can I fit all of my columns into the page, so that I do not need to slide and see them all together fitted?

2- Is there any possibility to change the color from one row to another? For example, I have 10 rows and I want to make the color of (1,3,5,...) gray and (2,4,6,...) dark-gray.

**Comment by melohagan** [comment]
Hi @ozannsk 

To fit all of the columns on the screen, you can customise the column widths: https://docs.budibase.com/docs/table
Where you see 'value' enter a percentage. So if you had three columns you could do 30% each. 

If your table contains 'View' links or other row buttons, the total value percentage you can use for your column widths will be less than 100 because those buttons and links will take up some space in the table as well.

As for the striped rows, there is an issue open here to add more flexibility to table formatting: https://github.com/Budibase/budibase/issues/3273
